runtime/rastrace/method_trigger.c

```diff
@@ -547,6 +546,50 @@ decimalString2Int(J9PortLibrary* portLibrary, const char *decString, I_32 signed
 	return num;
 }
 
+/**************************************************************************
+ * name        - setMethodStrArgLength
+ * description - Set method string argument length
+ * parameters  - thr, trace options, atRuntime flag
+ * returns     - JNI return code
+ *************************************************************************/
+omr_error_t
+setMethodStrArgLength(J9JavaVM *vm, const char *str, BOOLEAN atRuntime)
+{
+#define MAX_STRING_LENGTH 128
+	PORT_ACCESS_FROM_JAVAVM(vm);
+	int value, length;
+	omr_error_t rc = OMR_ERROR_NONE;
+	const char *p;
```

Comment:
Declare one local at a time, and initialize each; `p` is not a very descriptive name, even `parm` would be better.
```c
	int value = 0;
	int length = 0;
	omr_error_t rc = OMR_ERROR_NONE;
	const char *parm = NULL;
```